test/test.ts

```diff
@@ -15,6 +15,7 @@ import { Actor, duplicateActor } from '../scripts/libraries/actor';
 import { TestDependencies } from './test-dependencies';
 import { generateDai } from '../scripts/seed/seed-maker';
 import { resetMaker } from '../scripts/seed/reset-maker';
+import { IOffChainTransaction } from "@zoltu/ethereum-types/output-node";
```

Comment:
This should be `import { IOffChainTransaction } from '@zoltu/ethereum-types'`.

Reply:
ahh, IDE burned me

Reply:
Yeah, it does that to me sometimes to, I don't know why.